lib/utils/version-route.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
-
-const pkg = require('../../package.json');
+import path from 'path';
```

Comment:
We haven't adopted the use of `import` yet. I'd like to do it all at once, instead of having some `require` and some `import`. Mind changing to `const path = require('path');`?